fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/util/CommonUtil.java

```diff
@@ -116,7 +116,18 @@ public static void printUsage() {
 
         // Drops a Tenant
         ps.println("--drop-tenant tenantName");
-        ps.println(" * drops the tenant given the tenantName");
+        ps.println(" * (phase 1) drops the tenant given the tenantName");
```

Comment:
it's a little bit strange that "drop-tenant" doesn't really drop the detached tables. maybe we should change it to "detach-tenant"?

Reply:
It does try to drop the tables. It could potentially succeed in the first call if Db2 were quick enough with its async processes.